package.json

```diff
@@ -16,5 +17,8 @@
 		"lerna": "~5.0.0",
 		"lint-staged": "^12.4.2",
 		"prettier": "2.6.2"
+	},
+	"devDependencies": {
+		"eslint": "^8.19.0"
```

Comment:
Why did you have to add this package in the root package.json?

Reply:
It was added when running `npm run bootstrap`